lib/srv/db/mysql/autousers.go

```diff
@@ -464,25 +515,39 @@ func readDeleteUserResult(res *mysql.Result) string {
 	return string(res.Values[0][0].AsString())
 }
 
+func getCreateProcedureCommand(conn *clientConn, procedureName string) (string, bool) {
+	if conn.isMariaDB() {
+		command, found := mariadbProcedures[procedureName]
+		return command, found
+	}
+	command, found := mysqlProcedures[procedureName]
+	return command, found
```

Comment:
nit:
```suggestion
	if conn.isMariaDB() {
		return mariadbProcedures[procedureName]
	}
	return mysqlProcedures[procedureName]
```

Reply:
seems go doesn't like this. will keep as it is.